solver/src/solver/paraswap_solver.rs

```diff
@@ -60,37 +63,80 @@ impl ParaswapSolver {
 #[async_trait::async_trait]
 impl SingleOrderSolving for ParaswapSolver {
     async fn settle_order(&self, order: LimitOrder) -> Result<Option<Settlement>> {
+        let transaction: TransactionBuilderResponse;
+        let price_response: PriceResponse;
+        let amount: U256;
+        loop {
```

Comment:
Should we bound this loop?

Reply:
Yea great idea. I would think at most 1 or 2 retries is enough... but then we will have to spit out the same PriceChange error if its still happening.